src/main/java/com/bugsnag/android/ExceptionHandler.java

```diff
@@ -47,18 +51,30 @@ public ExceptionHandler(UncaughtExceptionHandler originalHandler) {
         this.originalHandler = originalHandler;
     }
 
+    @Override
     public void uncaughtException(@NonNull Thread t, @NonNull Throwable e) {
+        boolean strictModeThrowable = strictModeHandler.isStrictModeThrowable(e);
+      
         // Notify any subscribed clients of the uncaught exception
-        for(Client client : clientMap.keySet()) {
-            client.cacheAndNotify(e, Severity.ERROR);
+        for (Client client : clientMap.keySet()) {
+
+            if (strictModeThrowable) { // add strictmode policy violation to metadata
+                String violationDesc = strictModeHandler.getViolationDescription(e.getMessage());
+                MetaData metaData = new MetaData(
+                    Collections.<String, Object>singletonMap(KEY_STRICT_MODE_VIOLATION, violationDesc));
```

Comment:
This invocation does not specify a tab for the metadata, so its appearing on its own "custom" tab. Since there might be other strictmode-related features we might want to surface in the future, we can instead put this on its own "StrictMode" tab with something like (assuming the appropriate constants for the tab and key names):

```diff
- MetaData metaData = new MetaData(
-     Collections.<String, Object>singletonMap(KEY_STRICT_MODE_VIOLATION, violationDesc));
+ MetaData metaData = new MetaData();
+ metaData.addToTab("StrictMode", "Violation", violationDesc);
```